telemetry/definitions/commonDefinitions.json

```diff
@@ -446,6 +467,26 @@
             "description": "Open to view status of an ECS Cluster",
             "metadata": [{ "type": "result" }]
         },
+        {
+            "name": "ecr_deployImage",
+            "description": "Called when deploying an image to ECR",
+            "metadata": [{ "type": "result" }, { "type": "regionId" }]
+        },
+        {
+            "name": "ecs_deployService",
+            "description": "Called when deploying a service to an ECS cluster",
+            "metadata": [{ "type": "result" }, { "type": "regionId" }, { "type": "ecsLaunchType" }]
+        },
+        {
+            "name": "ecs_deployTask",
+            "description": "Called when deploying a task to an ECS cluster",
+            "metadata": [
+                { "type": "result" },
+                { "type": "regionId" },
+                { "type": "ecsLaunchType" },
+                { "type": "scheduled" }
```

Comment:
what does scheduled mean?

Reply:
ECS tasks can be [scheduled](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/scheduling_tasks.html) as part of the deployment process.

(Line 134's description: "Whether or not the referenced item is scheduled (true) or not (false)")

Reply:
I think this should be optional, or not have it at all. What questions does this answer?

Reply:
When you deploy in VS, deploying ECS tasks and deploying scheduled ECS tasks are top level items, so it indicates how much each workflow is used.

Reply:
Tasks and Scheduled Tasks are top level Cluster entities, just like Services, so the "scheduled" field should be required.

![image](https://user-images.githubusercontent.com/39839589/95134298-7dbb4800-0717-11eb-804a-ea7e0efd1c9f.png)

Reply:
If it's a different top level thing, should it not be its own metric?

Reply:
It is using the same construct (Tasks), so I wouldn't think so.